config.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+
+def arg_parse():
+    parser = argparse.ArgumentParser(description='PyTorch DQN Training')
+
+    parser.add_argument('--savePrefix', type=str, help='the save directory of the running results')
+    parser.add_argument('--epsStart', type=float, help='the initial epsilon')
+    parser.add_argument('--rlmodel', type=str, default='None', choices=['DQN', 'DuelingDQN'], help='model of experiment (default: DQN)')
```

Comment:
Acting as a basic reference implementation, `DuelingDQN` is actually out-of-scope in this repo. Maybe we'd better remove all lines about `DuelingDQN` and just keep this repo simple?